src/Nethermind/Nethermind.Network/ForkInfo.cs

```diff
@@ -38,7 +38,7 @@ public static byte[] CalculateForkHash(ISpecProvider specProvider, long headNumb
                     break;
                 }
 
-                BinaryPrimitives.WriteUInt64BigEndian(blockNumberBytes, (ulong)transitionBlocks[i]);
+                BinaryPrimitives.WriteUInt64BigEndian(blockNumberBytes, (ulong)transitionBlocks[i].BlockNumber);
```

Comment:
QQ: what is current approach to fork_id? Do you know something more what is the plan for shanghai hardfork?

Reply:
Could not attend the meeting in Bogota but it is not clear yet. Waiting for official spec or geth implementation